pkgs/servers/home-automation/evcc/node-packages.nix

```diff
@@ -0,0 +1,17910 @@
+# This file has been generated by node2nix 1.9.0. Do not edit!
```

Comment:
18000 lines is a bit too much for a package I don't use but still have to download via nixpkgs.

Reply:
This would be a better solution: https://github.com/NixOS/nixpkgs/pull/128749

Reply:
Yeah, when though?